js/viz/core/utils.js

```diff
@@ -444,6 +444,50 @@ export function getVizRangeObject(value) {
     }
 }
 
+export function normalizeArcParams(x, y, innerR, outerR, startAngle, endAngle) {
+    let isCircle;
+    let noArc = true;
+    const angleDiff = roundValue(endAngle, 3) - roundValue(startAngle, 3);
+    if(angleDiff) {
+        if((abs(angleDiff) % 360) === 0) {
+            startAngle = 0;
+            endAngle = 360;
+            isCircle = true;
+            endAngle -= 0.01;
+        }
+
+        if(startAngle > 360) {
+            startAngle = startAngle % 360;
```

Comment:
Let's move 360 to a constant?

Reply:
Apart from the fact that it was a migration of the old working code to another location, then I have a question.

What do you understand when you see the numbers 180, 360, 720 and so on? I understand this in the context of any circles, that this is an angle, if such things are taken out in a separate explanation in terms of a variable, then it seems to me it can confuse more

Reply:
> What do you understand when you see the numbers 180, 360, 720 and so on?

I don't wanna say it's impossible to understand the meaning of this number now. But i need to waste a little brain resource to realise what this variable mean.

180 is not only a half on a full angle, it's also number of Fahrenheit degree between freezing point and boiling point, angry number from "Final destination", max result in darts and so on.

See https://eslint.org/docs/latest/rules/no-magic-numbers

Anyway, it's not so important now, so u can skip this note